lldb-eval/parser.cc

```diff
@@ -938,6 +966,7 @@ ExprResult Parser::ParseUnaryExpression() {
   }
 
   if (token_.is(clang::tok::kw_sizeof)) {
+    clang::Token token = token_;
```

Comment:
Nit: The source location can be calculated directly here.

Reply:
Done